src/settings-ui/Settings.UI/Flyout/ShellPage.xaml

```diff
@@ -8,7 +8,7 @@
     Loaded="Page_Loaded"
     mc:Ignorable="d">
 
-    <Grid>
+    <Grid Background="{ThemeResource AcrylicBackgroundFillColorDefaultBrush}">
```

Comment:
Would love to understand why this needs to be added? The window already handles the Acrylic base layer?

Reply:
Without this the flyout theme is updated (text becomes white, etc), however the grid itself does not seem to use the default acrylic background fill color and simply becomes a slightly different gray (as shown in the attached image.

![image](https://user-images.githubusercontent.com/61442448/220813958-864faec9-0399-4daf-8959-a9c3f0a881ef.png)

Reply:
That's because the theme of the window is not changed.. can't we do that?

There might be other things (like tooltips or flyouts) not adjusting to the proper theme if we don't update the window itself (I know we ran into this before woth Settings itself).

Reply:
@niels9001 The window theme was actually getting changed, however the flyout colors in the `AppsListPage.xaml` and `LaunchPage.xaml` did not have a definition in the dark resource dictionary. I have updated `ShellPage.xaml` to use the Windows system colors for light or dark mode accordingly, and now the flyout color scheme almost perfectly resembles the Win11 action center.

The themes should be updating just fine, it just so happens that the chosen brushes for the flyout did not have a dark mode equivalent.